hrm-service/service-tests/case-audits.test.js

```diff
@@ -27,14 +31,18 @@ const caseAuditsQuery = {
 };
 
 beforeAll(async () => {
+  await proxiedEndpoints.start();
+  await proxiedEndpoints.mockSuccessfulTwilioAuthentication(workerSid);
```

Comment:
Would be possible to merge this with the server start, and use some config parameters? Just asking, no need to change anything here!

Reply:
We can do - I was thinking we might be mocking out other endpoints in other tests in the future and we will want to pick and choose which endpoints we mock & how, separately to starting the server